drools-wb-screens/drools-wb-globals-editor/drools-wb-globals-editor-client/src/main/java/org/drools/workbench/screens/globals/client/editor/AddGlobalPopupView.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.globals.client.editor;
+
+import java.util.List;
+
+import org.uberfire.client.mvp.UberElement;
+
+public interface AddGlobalPopupView extends UberElement<AddGlobalPopupView.Presenter> {
+
+    interface Presenter {
+
+        void validate();
+
+        void onAddButtonClicked();
+
+        void onCancelButtonClicked();
+    }
```

Comment:
We might have to refresh the MVP coding conventions to the team at some point. I see views in presenter implementations, presenter interfaces in View interfaces. 

Personally I don't care that much about the above. I think we agreed to drop the Presenter interfaces and I have to say I can't remember where we agreed to put the View. My guess is in a separate class :)

What I do care is the use of "on" on the start of each method that is called from view. So here my suggestion is that you could also add that to the `validate()` method. Just that nobody calls if from anywhere else. Unless that is meant to be called from other presenters/parts of the code than `AddGlobalPopupViewImpl`.